src/bo4e/enum/landescode.py

```diff
@@ -1,32 +1,12 @@
-import iso3166
+from enum import Enum
+from iso3166 import countries
 
+"""
+Der ISO-Landescode als Enumeration.
+"""
 
-class Landescode:
-    """
-    Der ISO-Landescode.
-    """
+alpha2codes = {}
+for c in countries:
+    alpha2codes[c.alpha2] = c.alpha2
```

Comment:
```suggestion
alpha2codes = {c.alpha2: c.alpha2 for c in countries}
```

Dictionary/List Comprehensions werden irgendwann unleserlich aber hier fände ich es ok.